pkg/compactor/compactor_test.go

```diff
@@ -39,7 +39,9 @@ var _ = Describe("Running Compactor", func() {
 		embeddedEtcdQuotaBytes int64  = 8 * 1024 * 1024 * 1024
 		snapshotTimeout               = 30 * time.Second
 		defragTimeout                 = 30 * time.Second
-		needDefragmentation           = true
+		// keep SleepForScrape 0 so that tests are faster
```

Comment:
This value is set to 0 primarily because there is no necessity for scraping metrics. Tests being faster is a side benefit of this. Please mention this.

Or better yet, simply remove this variable and directly set the field to `0` when creating `compactorConfig` in the test.

Reply:
Done